packages/mobile/src/features/videos/components/Movies/SlaiderNormal.tsx

```diff
@@ -23,12 +23,12 @@ export class SlaiderNormal extends React.Component<any,SlaiderNormalS>{
     this.state = {
       listVideos : []
     }
-    this.Separator = this.Separator.bind(this);
-    this.ImgeSlaider = this.ImgeSlaider.bind(this);
+    this.separator = this.separator.bind(this);
```

Comment:
jak definujesz metode jako arrow function to nie musisz uzywac bind

Reply:
faktycznie, nie musze ich bindować :)